### DISCUSSION TITLE
Documentation for convert function of Convert Class

### DISCUSSION THREAD

**Discussion by stc1988** [Moddable-OpenSource/moddable#637] [Q&A] [answered]
Documentation is [convert](https://github.com/Moddable-OpenSource/moddable/blob/e9742b49d5132f2723d3a5fca45b65cf3818d01a/documentation/commodetto/commodetto.md#convertsrc-dst) , but implementation is [process](https://github.com/Moddable-OpenSource/moddable/blob/e9742b49d5132f2723d3a5fca45b65cf3818d01a/modules/commodetto/commodettoConvert.js#L25).

Is the documentation wrong?

**Comment by phoddie** [comment, marked as answer]
Yes, it should be `process`. Thank you for the report. The change will be included in the next SDK update.

**Reply by stc1988** [reply]
I confirmed https://github.com/Moddable-OpenSource/moddable/commit/6877030f6e747479c1d16eaf21e0bd24ae4dfb7a . Thank you.
